modules/launcher/src/main/java/org/mule/runtime/module/launcher/log4j2/ArtifactAwareContextSelector.java

```diff
@@ -47,7 +49,8 @@
  */
 class ArtifactAwareContextSelector implements ContextSelector, Disposable {
 
-  static final StatusLogger logger = StatusLogger.getLogger();
+  static final StatusLogger STATUS_LOGGER = StatusLogger.getLogger();
```

Comment:
You used status logger because that is the name of the class that owns it... but that is not clear. I prefer just logger as before, otherwise use a name that describe why it must be used from outside this class